[FLINK-19381][docs] Fix docs for savepoint relocation (#13488)
apache/flink: docs/ops/state/savepoints.md

/savepoints/savepoint-:shortjobid-:savepointid/...
{% endhighlight %}

> After <a href="https://issues.apache.org/jira/browse/FLINK-5763">FLINK-5763</a> savepoint is self-contained and relocatable now, you can move the file and restore from any location.

Reviewer: ```suggestion
Since Flink 1.11.0, savepoints are self-contained and relocatable (see <a href="https://issues.apache.org/jira/browse/FLINK-5763">FLINK-5763</a> for details). You can move the file and restore from any location.
```

<div class="alert alert-info">

> <strong>Note: </strong>savepoint relocatable does not support taskowned state(such as GenericWriteAhreadLog sink) and <a href="{% link ops/filesystems/s3.md %}#entropy-injection-for-s3-file-systems">entropy injection</a> currently.

Reviewer: ```suggestion
<strong>Note: </strong>relocatable savepoints currently do not support task-owned state(such as `GenericWriteAhreadLog` sink) or <a href="{% link ops/filesystems/s3.md %}#entropy-injection-for-s3-file-systems">entropy injection</a>.
```

Reviewer: Can you actually add links to Jira tickets for these two points?

Author: @NicoK Do you mean adding the two Jira tickets which introduce these two features(FLINK-4939 for GenericWriteAheadSInk and FLINK-9061 for entropy-injection)?
